cob_navigation_config/cob4-5/costmap_common_params.yaml

```diff
@@ -0,0 +1,28 @@
+#BEGIN VOXEL STUFF
+#map_type: voxel
+map_type: costmap
+#origin_z: 0.0
+#z_voxels: 2
+#z_resolution: 0.23
+
+unknown_cost_value: 0
+unknown_threshold: 2
+mark_threshold: 0
+publish_voxel_map: true
+#END VOXEL STUFF
+
+transform_tolerance: 0.4
+obstacle_range: 2.5
+raytrace_range: 3.0
+inflation_radius: 0.55
+footprint: [[0.4,0.0],[0.35,0.35],[-0.2,0.4],[-0.45,0.0],[-0.2,-0.4],[0.35,-0.35]]
+
+# BEGIN VOXEL STUFF
+observation_sources: laser_scan_front laser_scan_left laser_scan_right
+laser_scan_front: {sensor_frame: base_laser_front_link, topic: /base_laser_front/scan, data_type: LaserScan, marking: true, clearing: true}
+
+laser_scan_left: {sensor_frame: base_laser_left_link, topic: /base_laser_left/scan, data_type: LaserScan, marking: true, clearing: true}
+
+laser_scan_right: {sensor_frame: base_laser_right_link, topic: /base_laser_right/scan, data_type: LaserScan, marking: true, clearing: true}
```

Comment:
this is the old costmap params layout, please use the new one: https://github.com/ipa320/cob_navigation/blob/indigo_dev/cob_navigation_config/cob4-1/costmap_common_params.yaml